pkg/kubelet/types/pod_update.go

```diff
@@ -159,21 +159,20 @@ func IsCriticalPod(pod *v1.Pod) bool {
 	return false
 }
 
-// Preemptable returns true if preemptor pod can preempt preemptee pod:
-//   - If preemptor's is greater than preemptee's priority, it's preemptable (return true)
-//   - If preemptor (or its priority) is nil and preemptee bears the critical pod annotation key,
-//     preemptee can not be preempted (return false)
-//   - If preemptor (or its priority) is nil and preemptee's priority is greater than or equal to
-//     SystemCriticalPriority, preemptee can not be preempted (return false)
+// Preemptable returns true if preemptor pod can preempt preemptee pod
+// if preemptee is not critical or if preemptor's priority is greater than preemptee's priority
 func Preemptable(preemptor, preemptee *v1.Pod) bool {
+	if !IsCriticalPod(preemptee) {
```

Comment:
Shouldn't this be:
```
if IsCriticalPod(preemptor) && !IsCriticalPod(preemptee)
```

Reply:
Technically, yes.  Although we only call this function on preemptor pods that are critical.  Ill update it.

Reply:
done